### DISCUSSION TITLE
Fade effect when transitioning into a new cut?

### DISCUSSION THREAD

**Discussion by Kuhllen** [mifi/lossless-cut#2019] [Ideas]
Not sure if this feature already exists and I can't find it.

When merging clips. There should be an option for "fade into next clip. So that you don't have that sudden jump from clip to clip. I'd assume you could make this feature by allowing the program to insert some black box over the video as it goes into the next clip and then its opacity drops to 0 gradually to give a "fade" effect into the next clip?

**Comment by mifi** [comment]
https://github.com/mifi/lossless-cut/blob/master/issues.md#faq
